code/game/gamemodes/wizard/wizard.dm

```diff
@@ -19,44 +19,46 @@
 
 	var/finished = 0
 
+	var/const/players_per_wizard = 10
 	var/const/waittime_l = 600 //lower bound on time before intercept arrives (in tenths of seconds)
 	var/const/waittime_h = 1800 //upper bound on time before intercept arrives (in tenths of seconds)
+	var/wizard_amount= 1
 
 	can_be_mixed = TRUE
 
 /datum/game_mode/wizard/announce()
 	to_chat(world, "<B>The current game mode is - Wizard!</B>")
-	to_chat(world, "<B>There is a <span class='danger'>SPACE WIZARD on the station. You can't let him achieve his objective!</span>")
+	to_chat(world, "<B>There are a number of <span class='danger'>SPACE WIZARDS on the station. You can't let them achieve their objective!</span>")
```

Comment:
>There are a number

I'd recommend saying something like "There are several"

Also "objecive" should be plural
It probably already should have been but especially now